docs/WorkFlow.md

```diff
@@ -0,0 +1,74 @@
+# Workflow of Mentorship System Android App
+
+Mentorship System is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period.
+
```

Comment:
add about splash screen and onboard screen

Reply:
Done

Reply:
Add the layout ss as well